ethereum/scripts/configure-beefy.ts

```diff
@@ -0,0 +1,55 @@
+require("dotenv").config();
+const hre = require("hardhat");
+let { ApiPromise, WsProvider } = require('@polkadot/api');
+let { bundle } = require("@snowfork/snowbridge-types");
+
+const relaychainEndpoint = process.env.RELAYCHAIN_ENDPOINT;
+
+async function main() {
+  const signer = await hre.ethers.getSigner()
+
+  const beefyDeployment = await hre.deployments.get("BeefyLightClient");
+
+  const validatorRegistryDeployment = await hre.deployments.get("ValidatorRegistry");
+  const validatorRegistryContract = await new hre.ethers.Contract(validatorRegistryDeployment.address,
+    validatorRegistryDeployment.abi);
+
+  const validatorRegistry = await validatorRegistryContract.connect(signer)
+
+  const relayChainProvider = new WsProvider(relaychainEndpoint);
+  const relaychainAPI = await ApiPromise.create({
+    provider: relayChainProvider,
+    typesBundle: bundle
```

Comment:
Its not necessary to provide snowbridge types if you're only querying the relay chain.

Reply:
fixed c4420d0